Add streamlined data policy (#27)
orcestra-campaign/book: orcestra_book/data_policy.md

# Data Policy

> ORCESTRA encourages free use and access to data without strict control. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.

Reviewer: ```suggestion
ORCESTRA encourages free use and access to data ~~without strict control~~. We trust in the integrity of the scientific community to honor the contributions of those who collected these data.
```

Reviewer 2: I agree, the words are more or less redundant. Removing them simplifies the language which is good.

The ORCESTRA data are a product of the collective efforts and funding of various independent research groups. It often represents years of work in developing, certifying, and operating instruments, often tied to specific research or Ph.D. projects. We ask users to respect the priority of the data collectors by coordinating use with the instrument Principal Investigators (PIs) and by clearly acknowledging the source of the data in any publications or presentations.

> 

Reviewer: Specific data may be bound by agreements associated with the platforms from which they were collected, for instance the data policies of the German Research Vessels, or HALO.

> 

Reviewer: ```suggestion
Specific data may be bound by agreements associated with the platforms from which they were collected, for instance the data policies of the German Research Vessels, or HALO.

```

Maybe we should add this one line too, although it is a bit contradictory. Anyhow, we probably need to include the data policies of the platforms somehow...

> For publications using ORCESTRA data, please include this acknowledgement:

Reviewer: ```suggestion
For publications using ORCESTRA data, please acknowledge that the data was collected during the ORCESTRA field campaign which has been supported by funding and support from and within Europe, North America, Barbados and Cape Verde. More specific funding should be recognized in the context of the use of specific data products.
```

> > "The data used in this publication was collected during the ORCESTRA field campaign and is available through [insert institute name]. ORCESTRA is supported by the European Space Agency (ESA), the Max Planck Society (MPG), the German Research Foundation (DFG), the German Aerospace Center (DLR), the German Federal Ministry for Education and Research (BMBF), and the US National Science Foundation (NSF)."

Reviewer: ```suggestion
```

Author: With the suggested changes, I would have no idea what to put into the acknowledgements 🤷‍♂️

Reviewer: How about

```suggestion
> "The data used in this publication was collected during the ORCESTRA field campaign and is available through [insert institute name]. ORCESTRA is an international initiative of institutes within Europe, North America, Barbados and Cape Verde.  [Add an additional sentence in case of specific fundings in the context of the use of specific data products]."
```
